.github/workflows/cancel.yml

```diff
@@ -0,0 +1,27 @@
+name: Cancel
+on:
+  workflow_run:
+    workflows: [
+        "Cygwin Build",
+        "Notify users based on issue labels",
```

Comment:
This is not triggered with push, but with adding label. Don't think should be on this list.